product_docs/docs/pgd/5/rel_notes/pgd_5.4.0_rel_notes.mdx

```diff
@@ -0,0 +1,67 @@
+| BDR       | 5.4.0   | bdr_init_physical: fixed debug logging of pg_ctl enabling output capture for debugging purposes.                                                                                                                                                                                                                |                  |
+| BDR       | 5.4.0   | Fix assertion failure when TargetColumnMissing conflict occurs in a Group Commit transaction.                                                                                                                                                                                                                   |                  |
+| BDR       | 5.4.0   | Fix detection of UpdateOriginChange conflict to be more accurate.                                                                                                                                                                                                                                               |                  |
+| BDR       | 5.4.0   | Support timeout for normal Group Commit transaction.                                                                                                                                                                                                                                                            |                  |
+| BDR       | 5.4.0   | Fix error handling in writer when there are lock timeouts or conflicts or deadlocks with and without group commit transactions.                                                                                                                                                                                 |                  |
+| BDR       | 5.4.0   | Handle failure of source of synchronization and reconciliation coordinator.                                                                                                                                                                                                                                     |                  |
+| BDR       | 5.4.0   | Reconcile correctly when origin node fails before sending prepare to any node.                                                                                                                                                                                                                                  |                  |
+| BDR       | 5.4.0   | Apply commit scope timeout to reconciliation.                                                                                                                                                                                                                                                                   |                  |
```

Comment:
```suggestion
```
This is just for feature we introduce in this version, so nothing changed for customer.